lib/menu.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'order'
+
+class Menu
+
+  attr_reader :menu
+
+  def initialize
+    @menu = [{ :name => "Risotto", :price => 7.99 },
+      { :name => "Large Pizza", :price => 12.99 },
+      { :name => "Fried Chicken", :price => 6.99 },
+      { :name => "Green Salad", :price => 5.99 },
+      { :name => "Pasta", :price => 8.99 },
+      { :name => "Vegetable Curry", :price => 6.99 },
+      { :name => "Rice", :price => 1.99 },
+      { :name => "Steamed Vegetables", :price => 1.99 },
+      { :name => "Acai Bowl", :price => 3.99 }]
+  end
+
+  def display_menu
+    menu.each_with_index do |item, index|
+      puts "#{index + 1}. #{item[:name]}, Price: £#{item[:price]}"
+    end
+  end
+
+  def display_order
+    order_total = 0
+    puts "Order summary"
+    @new_order.orders.each do |item|
```

Comment:
I would consider give item -1 semantic value by puting it into a var like:
`user_choice = item - 1`